src/components/logs/app/index.html

```diff
@@ -0,0 +1,150 @@
+            </div>  
+            <vscode-checkbox id="follow-chk"></vscode-checkbox>
+
+            <div class="tooltip">Show timestamp:
+                <span class="tooltiptext">Include timestamps on each line in the log output.</span>
+            </div>  
+            <vscode-checkbox id="timestamp-chk"></vscode-checkbox>
+
+            <div class="tooltip">Since:
+                <span class="tooltiptext">Only return logs newer than a relative duration like 5s, 2m, or 3h. Defaults to all logs.</span>
+            </div>          
+            <div style="width: 50px; display: inline-block">
+                <vscode-inputbox id="since-input" value="0"></vscode-inputbox>
+            </div>
+            <vscode-select id="since-select" class="w-80">
+                <vscode-option value='' selected>&nbsp;</vscode-option>
+                <vscode-option value='s'>seconds</vscode-option>
+                <vscode-option value='m'>minutes</vscode-option>
+                <vscode-option value='h'>hours</vscode-option>
+            </vscode-select>
+
+            <div class="tooltip ml-10">Tail:
+                <span class="tooltiptext">Lines of recent log file to display. Defaults show all log lines.</span>
+            </div>
+            <div style="width: 50px; display: inline-block">
+                <vscode-inputbox id="tail-input" value="-1"></vscode-inputbox>
+            </div>
+            <div class="float-right mr-50">
+                <vscode-button id="runBtn">Run</vscode-button>
+                <vscode-button id="stopBtn" class="display-none">Stop</vscode-button>
+                <vscode-button id="clearBtn" class="display-none">Clear</vscode-button>
+                <vscode-button id="resetBtn">Reset</vscode-button>                
+            </div>
+            <div class="tooltip ml-10">To terminal:
+                <span class="tooltiptext">Redirect logs to terminal.</span>
+            </div>  
+            <vscode-checkbox id="terminal-chk"></vscode-checkbox>
+        </div>
+        <div class="mt-5">
+            <span>Filter:</span>
+            <vscode-select id="filter-select" class="w-130">
+                <vscode-option value='all' selected>all</vscode-option>
+                <vscode-option value='include'>that match</vscode-option>
+                <vscode-option value='exclude'>that don't match</vscode-option>
+                <vscode-option value='after'>after match</vscode-option>
+                <vscode-option value='before'>before match</vscode-option>
+            </vscode-select>
+            <div style="width: 350px; display: inline-block">
+                <vscode-inputbox placeholder="Enter your key words" id='filter-input'></vscode-inputbox>
+            </div>
+            
+            <div class="tooltip ml-10">Wrap lines:
+                <span class="tooltiptext">Long words should be broken and wrap onto the next line.</span>
```

Comment:
The tooltip seems at odds with the caption.  Does this control whether lines wrap at all, or is the idea that lines _always_ wrap and this merely controls _where_ they are allowed to break?

Reply:
I'm a bit confused by your question but by answering it it should be the second option.
Let me explain. This checkbox changes the css rule used to display logs, `white-space: pre` or `white-space: pre-wrap`. 
In both cases white-spaces and line-break are honoured. The difference is that in the first case (checkbox not checked and `white-space: pre` rule used) we don't care if the lines are wider than the parent (by simplifying the vscode webview in our case). We just follow the actual text, when there is a new-line, a new-line is printed otherwise everything will go in one row.
![image](https://user-images.githubusercontent.com/49404737/115364051-4861dd80-a1c3-11eb-867c-15b2e493786a.png)

In the second case (checkbox checked and `white-space: pre-wrap` rule used), we still honour white-space and break-line from the original text but we also impose that a line will not be wider than the parent. In this way the logs are fully visible in our webview and one doesn't need to scroll horizontally.
![image](https://user-images.githubusercontent.com/49404737/115364243-7cd59980-a1c3-11eb-8e31-a8313ccff167.png)

So this checkbox just control where a line is allowed to break, somehow. Hopefully this answer your question.

By reading the tooltip maybe i got what your doubt, we don't break words but lines??
Feel free to edit it by yourself or to propose a better tooltip as i'm kinda lost here. Thank you!